### ISSUE TITLE
opening JFB or MHC besides a bible text freezes bibletime [BUG] 

### ISSUE THREAD

**Issue by HinzundKunz** [bibletime/bibletime#417]
**Bug description**
I was trying to open JFB besides a bible text (Luther 1912), and bibletime froze. I had to force close it.

**Steps to reproduce**
1. open a bible text
2. open another bible text in parallel view
3. change the second one into a commentary, in my case JFB or MHC.
4. with JFB or MHC, bibletime freezes. other commentaries don't freeze bibletime (e.g. Kings commentary, or german ones)

**Expected behavior**
the commentary should be displayed besides the bible text.

**System information:**
 Operating System: KDE neon 5.26
KDE Plasma Version: 5.26.2
KDE Frameworks Version: 5.99.0
Qt Version: 5.15.7
Kernel Version: 5.15.0-52-generic (64-bit)
Graphics Platform: X11
Processors: 6 × AMD Ryzen 5 4500U with Radeon Graphics
Memory: 15.0 GiB of RAM
Graphics Processor: AMD RENOIR
Manufacturer: Acer
Product Name: Aspire A515-44G
System Version: V1.12

**Comment by jaakristioja** [comment]
I can confirm this with BibleTime 3.0.3. This is a duplicate of #396 which was fixed by ebdffe70ac6f5a8670138c325d7b238624851208 in the master branch. We have not yet decided whether there will be a 3.0.4 release with this fix backported.
